go.sum

```diff
@@ -116,6 +116,7 @@ github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4/go.mod h1:ybxpYRF
 github.com/aliyun/alibaba-cloud-sdk-go v0.0.0-20190412020505-60e2075261b6/go.mod h1:T9M45xf79ahXVelWoOBmH0y4aC1t5kXO5BxwyakgIGA=
 github.com/aliyun/alibaba-cloud-sdk-go v0.0.0-20190620160927-9418d7b0cd0f h1:oRD16bhpKNAanfcDDVU+J0NXqsgHIvGbbe/sy+r6Rs0=
 github.com/aliyun/alibaba-cloud-sdk-go v0.0.0-20190620160927-9418d7b0cd0f/go.mod h1:myCDvQSzCW+wB1WAlocEru4wMGJxy+vlxHdhegi1CDQ=
+github.com/aliyun/alibaba-cloud-sdk-go v0.0.0-20190808125512-07798873deee h1:NYqDBPkhVYt68W3yoGoRRi32i3MLx2ey7SFkJ1v/UI0=
```

Comment:
do we need this? is this a transient dependency ?

Reply:
Yup. It looks like it:
```shell
$ go mod graph
...
github.com/go-acme/lego/v3@v3.1.0 github.com/aliyun/alibaba-cloud-sdk-go@v0.0.0-20190808125512-07798873deee
github.com/hashicorp/vault@v1.4.0 github.com/aliyun/alibaba-cloud-sdk-go@v0.0.0-20190620160927-9418d7b0cd0f
github.com/go-acme/lego/v3@v3.2.0 github.com/aliyun/alibaba-cloud-sdk-go@v0.0.0-20190808125512-07798873deee
github.com/hashicorp/vault-plugin-auth-alicloud@v0.5.4 github.com/aliyun/alibaba-cloud-sdk-go@v0.0.0-20190412020505-60e2075261b6
github.com/hashicorp/go-kms-wrapping@v0.0.0-20191129225826-634facde9f88 github.com/aliyun/alibaba-cloud-sdk-go@v0.0.0-20190620160927-9418d7b0cd0f
github.com/hashicorp/go-kms-wrapping@v0.5.1 github.com/aliyun/alibaba-cloud-sdk-go@v0.0.0-20190620160927-9418d7b0cd0f
github.com/hashicorp/vault-plugin-secrets-alicloud@v0.5.4 github.com/aliyun/alibaba-cloud-sdk-go@v0.0.0-20190620160927-9418d7b0cd0f
```